deployment/README.md

```diff
@@ -0,0 +1,92 @@
+# GCP Project Setup (WIP)
+
+Ensure the `GOOGLE_CLOUD_PROJECT` environment variable is exported for the project:
+```bash
+export GOOGLE_CLOUD_PROJECT=<PROJECT_ID>
+```
+The official project id of OSV is `oss-vdb`.
+
+## Terraform
+
+In this directory (`/deployment`):
+
+Initialise terraform:
+```bash
+terraform init
+```
+
+Plan and apply required project infastructure:
+```bash
+terraform plan
```

Comment:
Can you add some more detail on what `plan` does and what we should be doing with the output of that? Is this just to verify that the things it will do look sane, before actually applying them?

Reply:
If there's any good outlinks you can include at these steps to help an unfamiliar reader understand the "why" of these that would be helpful too.

Reply:
> Is this just to verify that the things it will do look sane, before actually applying them?

Yeah. `plan` is not actually necessary to run, but it shows you what resources would be added / changed / deleted with the configuration.
The output of plan is also shown when you run `apply`, which prompts you to accept the proposed changes.

I'll convey this in the README.

> If there's any good outlinks you can include at these steps to help an unfamiliar reader understand the "why" of these that would be helpful too.

I don't know if I've seen any good links, but will add more "why" information.